src/org/omegat/Bundle.properties

```diff
@@ -282,7 +282,28 @@ TF_MENU_FILE_CLEAR_RECENT=&Clear Menu
 TF_MENU_FILE_IMPORT=Add &Files...
 TF_FILE_IMPORT_TITLE=Add Files to Source Folder...
 
-TF_MENU_WIKI_IMPORT=Add MediaWiki Pa&ge...
+TF_MENU_EXTERNAL_SERVICE_IMPORT=Import External Service Pa&ge...
```

Comment:
Although I understand the idea, I don’t think that the labels are very friendly from an end user pov.

We need to reflect on how we want to advertize "external services"...